packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -72,7 +122,7 @@ describe('quantic-sort', () => {
 
       describe('when loading default sort', () => {
         it('should work as expected', () => {
```

Comment:
Not from your changes but since we are already here please update this description, cause I hate this `should work as expected` because the goal of this description is to explain what's expected.
```suggestion
        it('should display sort options correctly and allow selection of options', () => {
```